docs/source/release/v6.9.0/Inelastic/New_features/35665.rst

```diff
@@ -0,0 +1 @@
+- Moved 'Data Analysis' interface from Indirect and Direct interface menus into Inelastic interface menu.
```

Comment:
It is possible to add a link to the Data Analysis documentation file by doing something like this:
```suggestion
- Moved :ref:`Data Analysis <interface-inelastic-data-analysis>` interface from Indirect and Direct interface menus into Inelastic interface menu.
```